src/slack.js

```diff
@@ -269,6 +269,15 @@ async function handleCSVUpload(event, bot, say) {
     text: `*Invoices summary*\n${formattedInvoices}`,
   })
 
+  if (!bot.sendOnSlack) {
+    await say({
+      channel: bot.channel,
+      as_user: true,
```

Comment:
I actually found this on https://api.slack.com/methods/chat.postMessage#markdown: 
> Please note that the as_user parameter may not be used by [new Slack apps](https://api.slack.com/authentication/basics). Read more about [Authorship](https://api.slack.com/methods/chat.postMessage#authorship) to understand how as_user works for classic Slack apps.

Reply:
so I guess we can remove it from all the calls now. should I take over this so you can focus on Bee?

Reply:
I would gladly do it after after finishing my current task, but it will probably take one more day until I'm done. You can take it if you want it earlier.